app/tests/spec/views/settings/emails.js

```diff
@@ -285,6 +292,48 @@ define(function (require, exports, module) {
           assert.equal(view.isPanelOpen(), false);
         });
       });
+
+      describe('can change email', () => {
+        const newEmail = 'secondary@email.com';
+        beforeEach(() => {
+          emails = [{
+            email: 'primary@email.com',
+            isPrimary: true,
+            verified: true
+          }, {
+            email: newEmail,
+            isPrimary: false,
+            verified: true
+          }];
+
+          return initView()
+            .then(function () {
+              // click events require the view to be in the DOM
+              $('#container').html(view.el);
+            });
+        });
+
+        it('can render', () => {
+          assert.equal(view.$('.email-address').length, 1);
+          assert.equal(view.$('.email-address .address').length, 1);
+          assert.equal(view.$('.email-address .address')[0].innerHTML, 'secondary@email.com');
+          assert.equal(view.$('.email-address .details .verified').length, 1);
+          assert.equal(view.$('.email-address .settings-button.warning.email-disconnect').length, 1);
+          assert.equal(view.$('.email-address .settings-button.warning.email-disconnect').attr('data-id'), 'secondary@email.com');
+          assert.equal(view.$('.email-address .settings-button.secondary.set-primary').length, 1);
+          assert.equal(view.$('.email-address .settings-button.secondary.set-primary').attr('data-id'), 'secondary@email.com');
+        });
+
+        it('can change email', (done) => {
+          $('.email-address .settings-button.secondary.set-primary').click();
+          setTimeout(() => {
```

Comment:
The inner function needs to be wrapped in a [TestHelpers.wrapAssertion](https://github.com/mozilla/fxa-content-server/blob/8fafad7f418419c3eea679e345301e03a43c790c/app/tests/lib/helpers.js#L67) to prevent any failures in this test from timing out.  That or convert the entire thing to be promise based:

```js

$('.email-address .settings-button.secondary.set-primary').click();
return p.delay(150).then(() => {
  assert.equal(account.get('email'), newEmail, 'account email updated');
});
```